Classes/Command/FluidTemplateProfileCommand.php

```diff
@@ -0,0 +1,162 @@
+<?php
+namespace NamelessCoder\CmsFluidPrecompilerModule\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Logger\ConsoleLogger;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use TYPO3Fluid\Fluid\Core\Rendering\RenderingContextInterface;
+
+/**
+ * Command to profile Fluid templates
+ */
+class FluidTemplateProfileCommand extends Command
+{
+    use FluidPrecompilerServiceTrait;
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Profile Fluid templates')
+            ->addArgument(
+                'extension',
+                InputArgument::REQUIRED,
+                'Extension key of the Fluid template'
+            )
+            ->addArgument(
+                'template',
+                InputArgument::REQUIRED,
+                'Path to the template'
+            );
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $logger = new ConsoleLogger($output);
+        $outputHelper = new SymfonyStyle($input, $output);
+
+        // Get arguments and options
+        $extension = $input->getArgument('extension');
+        $template = $input->getArgument('template');
+
+        $result = $this->getPrecompilerService()->profileTemplateFile($template, $extension);
+
+        // General information
+        $outputHelper->section('General');
+        $rows = [
```

Comment:
This looks almost entirely like `$rows = [$result]` would do the same; except for squashing the sections which could be done afterwards. Since `$result['sections']` doesn't get used below, you could simply do:

```php
$result['sections'] = count($result['sections']);
$outputHelper->table(['Info', 'Value'], [$result]);
```

To replace lines 52-88.